generator/explores/explore.py

```diff
@@ -105,6 +105,23 @@ def _get_default_channel(self, view: str) -> Optional[str]:
             return escape_filter_expr(default_value)
         return None
 
+    def _get_base_name_and_metric(
+        self, view_name: str, views: List[str]
+    ) -> Tuple[str, str]:
+        """
+        Get base view and metric names.
+
+        Returns the the name of the base view and the metric based on the
+        passed `view_name` and existing views.
+        """
+        split = view_name.split("__")
+        for index in range(len(split) - 1, 0, -1):
```

Comment:
I assume you're working backward until you find a matching view, and we need to do it this way because there might be more than one view that matches? Can you add a quick comment describing the logic here?

Reply:
Added an example to the function doc comment